.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Issue_Template
+about: Create a useable template format for guidance
+title: '[ISSUES]'
+labels: 'Issue'
+assignees: ''
+
+---
+
+**Description of the issue**
+Describe the issue and share important details such as the desired solution, error messages, and where the issue originates.
```

Comment:
Describe the issue and share important details such as domain (frontend, backend, or both), error messages, and where the issue originates or is seen. (just my suggestion not forcing you to make any changes).

Reply:
I think you're right about adding details like what domain the issue is from.